README.md

```diff
@@ -55,9 +55,38 @@ var jobDetails = client.getJobDetails(job.id);
 `jobDetails` will contain all information normally found in a successful response from
 our [Get Job](https://www.rev.ai/docs#operation/GetJobById) endpoint
 
+### Checking multiple files
+
+You can retrieve a list of transcription jobs with optional parameters
+
+```javascript
+var jobs = client.getListOfJobs();
```

Comment:
This is my fault but I committed a change directly to master earlier today and forgot to push the rebase of develop. All the methods in the docs should be awaited. So this should be `var jobs = await client.getListOfJobs();`